heron/instance/src/java/com/twitter/heron/instance/Slave.java

```diff
@@ -249,8 +249,8 @@ private void handleRestoreInstanceStateRequest(InstanceControlMsg instanceContro
     }
     if (request.getState().hasState() && !request.getState().getState().isEmpty()) {
       @SuppressWarnings("unchecked")
```

Comment:
yes, we need to suppress the cast of `State<Serializable, Serializable>`, java's type erasure the two parameter type of the `State` is lost.